Sources/PrimerSDK/Classes/User Interface/OAuth/WebViewController.swift

```diff
@@ -57,6 +57,21 @@ internal class WebViewController: PrimerViewController, WKNavigationDelegate {
     func webView(_ webView: WKWebView, decidePolicyFor navigationAction: WKNavigationAction, decisionHandler: @escaping (WKNavigationActionPolicy) -> Void) {
         log(logLevel: .info, message: "🚀 \(navigationAction.request.url?.host ?? "n/a")")
         
+        if var urlStr = navigationAction.request.url?.absoluteString, urlStr.hasPrefix("bankid://") == true {
+            let settings: PrimerSettingsProtocol = DependencyContainer.resolve()
+            if urlStr.contains("redirect=null"), let urlScheme = settings.urlScheme {
+                urlStr = urlStr.replacingOccurrences(of: "redirect=null", with: "redirect=\(urlScheme)")
```

Comment:
we may need to ascertain with BankID & Klarna that this will work in the future, or at least make sure we have a back up in case it doesn't work.

Reply:
cool sounds good. So long as we don't break anything.